src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/health/DatabaseConnectionHealthIndicator.scala

```diff
@@ -32,7 +32,7 @@ class DatabaseConnectionHealthIndicator @Inject() (val dbProvider: DatabaseProvi
     extends HealthIndicator
     with Repository {
   import api._
-  private val log = LoggerFactory.getLogger(this.getClass)
+  private val log = Logger(this.getClass)
```

Comment:
Why not to do it with LazyLogging trait ?

Reply:
These classes already had a logger provided by slick or another framework named `logger`, but with a different type (`SlickLogger` or something like that), so there was a clash, and it won't compile. It's the same reason why the values are called `log`, `serviceLogger`, etc., in the first place.